curriculum/challenges/english/25-front-end-development/workshop-storytelling-app/671fafbae959c048d82bac8d.md

```diff
@@ -141,12 +141,10 @@ const storyObj = {
 };
 
 function displayStory(genre) {
-    if (storyObj.hasOwnProperty(genre)) {
-
-    result.textContent = storyObj[genre].story;
-    storyContainer.style.borderColor = storyObj[genre].borderColor;
- 
-}
+  if (storyObj.hasOwnProperty(genre)) {
+  result.textContent = storyObj[genre].story;
+  storyContainer.style.borderColor = storyObj[genre].borderColor;
```

Comment:
The body of the if-statement should be indented from the if-statement itself (and its closing brace). I.e. you should indent these two lines by two more spaces.